package.json

```diff
@@ -101,13 +101,13 @@
     "react-native-image-picker": "^5.0.2",
     "react-native-image-size": "git+https://github.com/Expensify/react-native-image-size#6b5ab5110dc3ed554f8eafbc38d7d87c17147972",
     "react-native-modal": "^13.0.0",
-    "react-native-onyx": "1.0.36",
+    "react-native-onyx": "1.0.39",
     "react-native-pdf": "^6.6.2",
     "react-native-performance": "^4.0.0",
     "react-native-permissions": "^3.0.1",
     "react-native-picker-select": "git+https://github.com/Expensify/react-native-picker-select.git#84ee97dec11c2e65609511eb5a757d61bbeeab79",
     "react-native-plaid-link-sdk": "^10.0.0",
-    "react-native-quick-sqlite": "^5.0.3",
+    "react-native-quick-sqlite": "^8.0.0-beta.2",
```

Comment:
Why is this change required here?

Reply:
In the react-Native-Onyx, we just updated "react-native-quick-sqlite": "^8.0.0-beta.2" and this package is peerDependencies so we need update it in the App repo

Reply:
Yes, it is [this PR](https://github.com/Expensify/react-native-onyx/pull/235)